### DISCUSSION TITLE
Blank page after installation on Raspberry Pi

### DISCUSSION THREAD

**Discussion by Bambus89** [datenstrom/community#263] [Report a bug] [answered]
I have only an blank Site after installing and opening yellow.

I have apache2, mod rewrite enabled and php 5.6. But I cant get yellow running. Whats my fault?
Running it on Debian 8.

**Comment by markseuffert** [comment]
Not your fault. My guess is that you need to adjust your mod_rewrite server configuration. [See troubleshooting](https://datenstrom.se/yellow/help/troubleshooting).

What's the URL of your website?
Do you see any errors in the the web server logfile?

**Comment by Bambus89** [comment]
The URL is http://christiankuhn.org/ and i dont see any logs or failures... Mod Rewrite is on...
other cms systems which needs mod rewrite too works properly such as kirby or grav or wordpress... Only yellow doesnt...
`root@christiankuhn:/home/christian# a2enmod rewrite
Module rewrite already enabled`

![bildschirmaufnahme vom 2017-07-06 16 56 47](https://user-images.githubusercontent.com/29486808/27917395-25dbffba-626c-11e7-8f2a-53100a3fd87f.png)

![bildschirmaufnahme vom 2017-07-06 16 57 38](https://user-images.githubusercontent.com/29486808/27917428-3f473618-626c-11e7-9578-fd718ebbe4af.png)


What do you suggest?

**Comment by markseuffert** [comment]
Strange, I don't get any response from your web server.

Please contact me, if you want me to have a closer look.

**Comment by Bambus89** [comment]
Done

**Comment by markseuffert** [comment, marked as answer]
Thanks for reporting this bug. 

Apparently PHP doesn't allow it when a class that inherits from another is declared before the parent class. This works without problems on most systems, but not on a Raspberry Pi. Here PHP crashes and the Apache error log file shows `AH00052: child pid 1799 exit signal Segmentation fault (11)`.

I updated the Markdown parser. Should be working now.

**Comment by Bambus89** [comment]
New fresh Install of Yellow will work :) Thank you for this great Support :)
